lib/host-sp-messages/src/lib.rs

```diff
@@ -91,11 +140,21 @@ pub enum DecodeFailureReason {
     // Microoptimization: insert a dummy variant first, so we never serialize a
     // command value of `0` to make COBS's life slightly easier.
     _Unused,
-    CobsError,
-    HubpackError,
-    CrcFailure,
+    Cobs,
```

Comment:
Same note here: should we explicitly lock these to numerical values?

Reply:
Added in d4e84d9f